packages/core-utils/src/types.js

```diff
@@ -20,23 +27,65 @@ export const languageConfigType = PropTypes.shape({
   stopViewer: PropTypes.string
 });
 
+export const mapSymbolsType = PropTypes.arrayOf(
```

Comment:
This name seems too generic and IMO should instead suggest why the dockStrokeColor is a relevant attribute.